sstables/storage.cc

```diff
@@ -464,17 +461,9 @@ sstring s3_storage::make_s3_object_name(const sstable& sst, component_type type)
     return format("/{}/{}/{}", _bucket, sst.generation(), sstable_version_constants::get_component_map(sst.get_version()).at(type));
 }
 
-future<> s3_storage::ensure_remote_prefix(const sstable& sst) {
-    if (!_remote_prefix) {
-        co_await sst.manager().system_keyspace().sstables_registry_lookup_entry(_location, sst.generation());
```

Comment:
The system_kesypace::sstables_registry_lookup_entry() is now unused

Reply:
sure. will drop it along with `s3_storage::ensure_remote_prefix()` in this very commit.